### PR TITLE
Issue 26

### PR DESCRIPTION
**Pull request by martukas**

### CODE REVIEW

File: dataformats/multigrid/src/MapFile.cpp [hunk 1]
```diff
@@ -22,16 +23,20 @@ MapFile::MapFile(std::string filename) {
   }
 
   auto res = fstat(fd, &sb);
-  assert(res != -1);
```
Comment (mortenjc) [inline dataformats/multigrid/src/MapFile.cpp]: What's wrong with using assert() ? I think it is cleaner. Is it because 
it is compiled away wjen in Release mode?

File: prototype2/adc_readout/chopper_integration_test/CMakeLists.txt [hunk 2]
```diff
@@ -1,25 +1,18 @@
...
 )
 
-find_package(Threads REQUIRED)
-
-add_executable(AdcChopperIntgerationTest ${AdcChopperInt_SRC} ${AdcChopperInt_INC})
-
-target_include_directories(AdcChopperIntgerationTest PRIVATE ${ASIO_INCLUDE_DIR} "../" "../../common/")
-
-target_link_libraries(AdcChopperIntgerationTest ${LibRDKafka_LIBRARIES} ${CMAKE_THREAD_LIBS_INIT})
+create_executable(AdcChopperIntgerationTest)
```
Comment (SkyToGround) [inline prototype2/adc_readout/chopper_integration_test/CMakeLists.txt]: The correct solution here is to revert the changes to this file.
